lib/modules.nix

```diff
@@ -1329,7 +1332,6 @@ private //
     doRename
     evalModules
     filterOverrides
-    filterOverrides'
```

Comment:
This commit changes the interface of this effectively public function, and not exposing the symbol doesn't address that. Instead you can introduce a new internal `filterOverridesInternal`, while moving `filterOverrides'` to the `private` list above (so that we can deprecate it in the future).